### ISSUE TITLE
wdq lookup reference error

### ISSUE THREAD

**Issue by maxlath** [nichtich/wdq#14]
after following the apt-get/wget installation instructions, I tried the following command
`echo http://de.wikipedia.org/wiki/Argon | wdq lookup` (pasted from your [tweet](https://twitter.com/nichtich/status/654663931064008709))
which returned the following error:
`Can't call method "referenced_variables" on an undefined value at /usr/share/perl5/RDF/Query/Algebra/Service.pm line 255, <> line 1.`

Other commands such as `wdq -q '?c wdt:P361 wd:Q544'` seem to work just fine. Could this be that the master branch doesn't provide this functionality yet?

**Comment by nichtich** [comment]
Thanks for the bug report! Unfortunately I cannot reproduce error, at least [on a fresh Ubuntu 14.04](https://travis-ci.org/nichtich/wdq#L900). Which operating system do you use? Could you also tell what version of the packages you have installed?

```
$ perl -E 'use RDF::Query; say $RDF::Query::VERSION'
```

Further investigation requires a stack trace to show where the error originated from:

```
$ sudo apt-get install libcarp-always-perl
$ echo http://de.wikipedia.org/wiki/Argon | perl -MCarp::Always `which wdq` lookup
```

**Comment by maxlath** [comment]
Ubuntu: 12.04.5 LTS (yes, I'm waaay too late on updates)
libhttp-tiny-perl: 0.016-1
librdf-query-perl: 2.908-1

`$ perl -E 'use RDF::Query; say $RDF::Query::VERSION'`

```
Name "_p_librdf_world_s::OWNER" used only once: possible typo at /usr/share/perl5/RDF/Redland.pm line 49.
2.908
```

`$ echo http://de.wikipedia.org/wiki/Argon | perl -MCarp::Always `which wdq` lookup`

```
Can't call method "referenced_variables" on an undefined value at /usr/share/perl5/RDF/Query/Algebra/Service.pm line 255
    RDF::Query::Algebra::Service::referenced_variables('RDF::Query::Algebra::Service=ARRAY(0x62c1838)') called at /usr/share/perl5/RDF/Query/Algebra/GroupGraphPattern.pm line 255
    RDF::Query::Algebra::GroupGraphPattern::referenced_variables('RDF::Query::Algebra::GroupGraphPattern=ARRAY(0x62c11a8)') called at /usr/share/perl5/RDF/Query/Algebra/Project.pm line 343
    RDF::Query::Algebra::Project::referenced_variables('RDF::Query::Algebra::Project=ARRAY(0x62cc938)') called at /usr/share/perl5/RDF/Query.pm line 746
    RDF::Query::as_sparql('RDF::Query=HASH(0x62e3ef8)') called at /home/maxlath/scripts/wdq line 501
```

**Comment by nichtich** [comment]
I could not easily find the problem and planned to only support what's available at least in Ubuntu 14.04 LTS anyway, so I'm sorry for not fixing this bug.
